packs/data/familiar-abilities.db/luminous.json

```diff
@@ -0,0 +1,53 @@
+{
+    "_id": "ou91pzf9TlOnIjYn",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Your familiar glows with a comforting light. It sheds bright light in a 30-foot radius and dim light for the next 30 feet. It can suppress or resume this light by using an action, which has the concentrate trait.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "TokenLight",
+                "value": {
+                    "animation": {
+                        "intensity": 4,
+                        "speed": 1,
+                        "type": "sunburst"
+                    },
+                    "bright": 30,
+                    "color": "#ffff00",
+                    "dim": 60,
+                    "shadows": 0
+                }
+            }
```

Comment:
Could we get a screenshot of this light effect? We generally want to be very conservative with lights in the system in terms of colors / brightness. It tends to wash out a lot of tokens if we don't.  The current light spell for example definitely needs toned down but everburning torch is fine.

Reply:
<img width="2032" alt="Screen Shot 2022-06-14 at 11 42 23 AM" src="https://user-images.githubusercontent.com/55938242/173665169-8b8a2544-1c72-40b2-80ba-0f575906ba19.png">
I can definitely adjust it; when I zoom out, it's definitely hard to see the token, but easier when zoomed in.

Reply:
How about this one?
<img width="2032" alt="Screen Shot 2022-06-14 at 1 44 24 PM" src="https://user-images.githubusercontent.com/55938242/173685382-28ada34a-3539-48be-88a4-9bc907dcf444.png">
Or I can change the light effect completely:
<img width="2032" alt="Screen Shot 2022-06-14 at 1 47 48 PM" src="https://user-images.githubusercontent.com/55938242/173685903-a18315d1-6f94-4e2f-a306-e5562eff7ec0.png">